Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H

```diff
@@ -102,86 +105,87 @@ void UpdateMomentumPerezElastic (
     T_PR const g1s = ( T_PR(1.0) - vcDv1*inv_c2 )*gc*g1;
     T_PR const g2s = ( T_PR(1.0) - vcDv2*inv_c2 )*gc*g2;
 
-    // Compute s
-    T_PR s = 0;
+    // Compute variant relative velocity in cm frame
```

Comment:
```suggestion
    // Compute variant relative velocity in center-of-mass frame
```
Also: What is meant by `variant` here?

Reply:
It is to distinguish it from the invariant relative velocity defintion that is frame independent.

Reply:
Will make comment change.